index.d.ts

```diff
@@ -535,6 +540,11 @@ declare namespace tracer {
           enabled?: boolean
         }
       }
+
+      /**
+       * Configuration enabling LLM Observability. Enablement is superceded by the DD_LLMOBS_ENABLED environment variable.
+       */
+      llmobs?: llmobs.LLMObsEnableOptions
```

Comment:
this is on the `experimental` part of the enablement options, ie:
```javascript
const tracer = require('dd-trace').init({
  experimental: {
    llmobs: ...
  }
})

In the following change below this one in `index.d.ts`, it is also included on the base options, ie

```javascript
const tracer = require('dd-trace').init({
  llmobs: ...
})
```

Reply:
It depends whether it should be experimental or not for now.

Reply:
Did the llmobs team decide to keep this an experimental feature?